server/server.go

```diff
@@ -82,8 +88,8 @@ type Server struct {
 	dbPath      string
 	db          *db.DB
 	snapConf    *SnapshotConf
-	metrics     *ServerMetrics
-	diagnostics *ServerDiagnostics
+	metrics     *Metrics
```

Comment:
Indeed, it's a pretty common mistake. This one was also warned by `gometalinter`.